src/app/MessageDef/AttributeDatas.cpp

```diff
@@ -17,11 +17,11 @@
  */
 /**
  *    @file
- *      This file defines AttributeDataList parser and builder in CHIP interaction model
+ *      This file defines AttributeDatas parser and builder in CHIP interaction model
```

Comment:
nit: 'Data' is already a plural of datum, AttributeDatas -> AttributeData

Reply:
Done, rename to AttribtueDataIBs. thanks